pkg/views/workspace/info/view.go

```diff
@@ -59,21 +63,26 @@ func projectRender(project *serverapiclient.ProjectInfo) string {
 			Border(lipgloss.HiddenBorder()).
 			Rows(extensions...).Render()
 	}
-
-	projectView := "Project" + projectNameStyle.Render(*project.Name) + "\n" + "State  " + projectState + "\n" + extensionsTable
-
+	
+	projectView := "Project" + projectNameStyle.Render(*projectInfo.Name) + "\n" + "RepositoryUrl" + repositoryURLStyle.Render(*project.Repository.Url) + "\n" + "State  " + projectState + "\n" + extensionsTable
```

Comment:
One note from me.

I'd love to see the Repository be it's own field. We have more info on the repository than just the URL.

I'd love to see something like:
```
  Repository
     Url      (project.Repository.Url)
     Branch (project.Repository.Branch) (hidden if branch is not defined)
```

@idagelic do you agree?

Reply:
Yes, that sounds good